pkg/crossplane/metadata.go

```diff
@@ -68,5 +75,5 @@ func ConvertError(ctx context.Context, err error) error {
 		fmt.Errorf("%w (correlation-id: %q)", err, id),
 		http.StatusInternalServerError,
 		"internal-server-error",
-	).WithErrorKey(string(kErr.ErrStatus.Reason)).Build()
```

Comment:
What's the reason to remove this? I fear without the error-key it's not a valid OSB error.

Reply:
`kErr` is nil at this point